### ISSUE TITLE
Option to make Combinations required.

### ISSUE THREAD

**Issue by hitinder** [PrestaShop/hummingbird#404]
**Is your feature request related to a problem? Please describe.**
The problem starts when a customer decides to use the add the cart button from the home screen. This step uses default combinations to add items to the cart; therefore, store owners will get a wrong order and eventually have to return the item.

**Describe the solution you'd like**
To solve this problem, a feature should be introduced. In the BO if a default product combination is not selected then the customer should be forced to select combinations to ensure correct products are added to cart for an order.

In the below image - default style and size are selected. Even though the user wants a t-shirt for a girl with size L.

![image](https://user-images.githubusercontent.com/9841229/69502894-2eb45c00-0f25-11ea-9209-4f2c756b9b23.png)

**Comment by MatShir** [comment]
@Hlavtox @eshraw @Oksydan @NeOMakinG wdyt ?
Old feature request from the classic theme

**Comment by Hlavtox** [comment]
**Regarding the implementation:**
This feature is not implemented in classic theme and not supported properly on 1.7. If someone implements add to cart button from product list, he should use a condition, that if the product has combinations, show `Choose variant` button.

**Regarding default combination:**
Product always needs to have a default combination. Product not having default combination is a **database inconsistency**. Prestashop handles it well if `Add to cart` is clicked from product page, because there is a fallback mechanism that chooses default combination automatically.

TLDR - This issue is invalid and will be resolved by https://github.com/PrestaShop/hummingbird/issues/347 👍

**Comment by Oksydan** [comment]
@Hlavtox is right. 
We could improve by adding simpler quick view for products with combinations. 'Chose variant' could  open modal with only attribute selection, quantity input and add to cart button instead of linking to product page.

**Comment by MatShir** [comment]
> **Regarding the implementation:** This feature is not implemented in classic theme and not supported properly on 1.7. If someone implements add to cart button from product list, he should use a condition, that if the product has combinations, show `Choose variant` button.
> 
> **Regarding default combination:** Product always needs to have a default combination. Product not having default combination is a **database inconsistency**. Prestashop handles it well if `Add to cart` is clicked from product page, because there is a fallback mechanism that chooses default combination automatically.
> 
> TLDR - This issue is invalid and will be resolved by #347 👍

Make sense for me ! Let's close this issue then :)
